data/saas/config/mailchimp_config.yml

```diff
@@ -92,3 +96,41 @@ saas_config:
             {
               <masked_object_fields>
             }
+
+
+  consent_requests:
+    opt_in:
+      - method: POST
+        path: /allowlists/add
+        body: |
+          {
+            "email": "<email>"
+          }
+        client_config:
+          protocol: https
+          host: <transactional_domain>
+          authentication:
+            strategy: api_key
+            configuration:
+              body: |
+                {
+                  "key": "<transactional_api_key>"
+                }
+    opt_out:
+      - method: POST
+        path: /allowlists/delete
+        body: |
+          {
+            "email": "<email>"
+          }
+        client_config:
+          protocol: https
+          host: <transactional_domain>
+          authentication:
+            strategy: api_key
+            configuration:
+              body: |
+                {
+                  "key": "<transactional_api_key>"
+                }
+
```

Comment:
The primary distinction between the requests we save for the mailchimp consent requests are whether they are opt_in or opt_out requests, instead of read/update/delete like we have elsewhere.  Both put the api_key in the request body, with separate authentication than the default authentication.